### DISCUSSION TITLE
using commands to add/edit slots is not working

### DISCUSSION THREAD

**Discussion by GL17CH92** [TheIllusiveC4/Curios#286] [General]
trying to make some changes to the amount of certain curios slots for server mates on our ATM8 server and can't seem to get the anything from the commands page to work. Any tips/help is greatly appreciated

**Comment by TheIllusiveC4** [comment]
What have you tried and what was the result? Please be as detailed as possible so I'll know how best to help you.

**Reply by TGIBTheGuyInBlue** [reply]
im having the same issue the command works in singleplayer but if i go onto my server where i have OP the command wont except any input past /curios add
